tests/unit/providers/test_gemini_provider.py

```
@@ -915,6 +915,144 @@ def test_streaming_completion_with_multiple_tool_calls() -> None:
+    mock_response.candidates[0].finish_reason = Mock()
+    mock_response.candidates[0].finish_reason.value = "STOP"
+    mock_response.model_version = "gemini-2.5-flash"
+    mock_response.usage_metadata = None
+
+    second_chunk = _create_openai_chunk_from_google_chunk(mock_response, tool_call_counter)
+
+    assert second_chunk.choices[0].delta.tool_calls is not None
+    assert [tc.index for tc in second_chunk.choices[0].delta.tool_calls] == [1, 2]
+
+
+async def _async_iter_chunks(items: list[Mock]) -> Any:
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_async_iter_chunks`

(ANN401)